src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -25,6 +25,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * Main entry point to the RCV tabulator program.
```

Comment:
Main entry point to the RCV tabulator GUI.

Reply:
Done.

Reply:
Ugh, sorry, just realized this was `Main`. Your original way was better.

Reply:
Done.